packages/xrpl/src/client/index.ts

```diff
@@ -526,6 +533,33 @@ class Client extends EventEmitter<EventTypes> {
     }
   }
 
+  /**
+   * Get Definitions from server_definitions
+   *
+   * @returns void
+   * @example
+   * ```ts
+   * const { Client } = require('xrpl')
+   * const client = new Client('wss://s.altnet.rippletest.net:51233')
+   * await client.getDefinitions()
+   * console.log(client.definitions)
+   * ```
+   */
+  public async getDefinitions(): Promise<void> {
```

Comment:
Hello, where else is this method used? I couldn't find invocations of this method in this PR.